ui/components/app/signature-request-siwe/signature-request-siwe-tag/index.scss

```diff
@@ -0,0 +1,2 @@
+.signature-request-siwe-tag {
+}
```

Comment:
If there are no styles required we can remove this file

Reply:
Done